samples/net/sockets/echo_async/README.rst

```diff
@@ -78,6 +79,7 @@ To test:
 .. code-block:: console
 
     $ telnet 127.0.0.1 4242
+    $ telnet ::1 4242
```

Comment:
Only one of these telnet connections is going to work, right? (IPV4 vs 6)   You might want to write something like this instead as:

     $ telnet 127.0.0.1 4242     # use this for IPV4 
     $ telnet ::1 4242           # or this for IPV6